dataset.ttl

```diff
@@ -1071,7 +1134,9 @@ data:TexasSubprovince
     time:hasTime ischart:Carboniferous ;
     sdo:name "Texas Subprovince"@en .
 
-data:ThomsonOrogen a geof:Orogen ,
+data:ThomsonOrogen dct:hasPart data:NeoproterozoicLowerPaleozoicProvinces , data:NeoproterozoicLowerPaleozoicBasinTypes , data:MACROSSAN_IGNEOUS_PROVINCE , data:PAMA_IGNEOUS_PROVINCE , data:KENNEDY_IGNEOUS_PROVINCE , data:IRON_RANGE_PROVINCE , data:GREENVALE_PROVINCE , data:CHARTERS_TOWERS_PROVINCE , data:BARNARD_PROVINCE , data:WARBURTON_BASIN , dct:isPartOf data:TASMANIDES
+# TO CHECK THIS OUT WITH DAVE & BOB. check out all Tasmanides components #

```

Comment:
These comments are fine while you are working on the file John, but please go through and remove them once you have the answers you need, so that the 'final-final' version is clean